test/http_test.c

```diff
@@ -416,6 +416,66 @@ static int test_http_keep_alive_1_require_no(void)
     return test_http_keep_alive('1', 2, 0);
 }
 
+static int test_http_resp_hdr_limit(unsigned int limit)
```

Comment:
Should `limit` be a `size_t` to match the argument to `OSSL_HTTP_REQ_CTX_set_max_response_hdr_lines()`?